### ISSUE TITLE
Core Concept IdP Comments

### ISSUE THREAD

**Issue by robotdan** [FusionAuth/fusionauth-site#637]
https://fusionauth.io/docs/v1/tech/core-concepts/identity-providers/

Is this page different than  this one? https://fusionauth.io/docs/v1/tech/identity-providers/ 

The second one is more complete, perhaps we don't need the first one?

**Comment by mooreds** [comment]
I think we've chatted about this before. I think that the 'core concept' is definitely a subset of the '/tech/identity-providers/' section. 

I believe I argued then (and still believe) that identity providers are a core concept of using FusionAuth and are slightly nuanced compared to the traditional 'identity provider' concept in common IAM parlance. (Just like 'application' means a different thing in the FusionAuth context.)

So I want something for people to grab ahold of w/r/t Identity Providers that isn't the whole kit and kaboodle that is the '/tech/identity-providers/' section. 

Happy to revise this section or tweak it somehow, but I really think that identity providers are a core concept of FusionAuth and should be at least nodded to here.
